### ISSUE TITLE
Hyperlinks with a #

### ISSUE THREAD

**Issue by Kuhrt** [Tizra/Tizra-Customer-Tracker#260]
We have some hyperlinks on a couple of our pages that have the # sign in them. After looking through some documentation I saw that there are some different things you can do with hyperlinks by adding #. The problem is that it's just part of the link so when you click it, it tries to find a link on our tizra page with all the text after the # sign. http://ccfcs.tizrapublisher.com/types_investments_dollarsandsense/2 and the Treasury Bills link is an example you can see. Is there a way to get it to link with a # sign in the link?

**Comment by daviddurand** [comment]
The reader does not work with '#' links as targets. This is something that we will be doing in the future, but it's tricky, as we have to build the targets and the browser scrolling/highlighing, unlike the case with pure HTML.

I have documented creating anchors (links that start with #) in the wiki, but you will see that we don't claim that they work yet. In fact, they don't work, and can even cause problems if you link into the reader with a fragment identifier on the link.

We are waiting on improvements in jQuery Mobile (you can read about its use of the "hash" in navigation on their web site).

You can of course use links to URLs to external sites that include fragment identifiers (#-strings at the end of complete URLS). Tizra does support a few special forms of fragment identifier when processing a PDF, as PDF does not let us otherwise attach things to links to control target windows, rel attributes, and the like. If you turn off the reader, links that have fragment identifiers won't cause trouble, but they won't do anything special either.

**Comment by Kuhrt** [comment]
Yeah, I understand that you can't use anchors within the PDF. Our link is an external site with a fragment identifier. Here's the actual link on the page that I linked earlier. http://www.investopedia.com/video/play/treasury-bill/#axzz29V200QLt when you click the link, it links to http://ccfcs.tizrapublisher.com/types_investments_dollarsandsense/1#axzz29V200QLt which is nothing of course. There's several other links in the document as well and only the ones with a # mess up. I guess I'm not sure what to turn off in the reader to get them to not cause trouble

**Comment by daviddurand** [comment]
OK, now I've gotcha... This is a bug that I'll look into.

**Comment by Kuhrt** [comment]
Arlene just let me know that the links she had posted changed their url, so they no longer have the # in them. I don't think we have any others that were this way, so it's not an issue for us anymore. Thanks!
